rotkehlchen/accounting/structures/evm_event.py

```diff
@@ -156,6 +156,7 @@ def serialize(self) -> dict[str, Any]:
             'counterparty': self.counterparty,
             'product': self.product.serialize() if self.product is not None else None,
             'address': self.address,
+            'extra_data': self.extra_data,
```

Comment:
why did you add this here? The extra data are not supposed to be serialized for the api ..

Reply:
ah wait no, you need it for editing probably